kymatio/scattering2d/backend/tensorflow_backend.py

```diff
@@ -16,11 +16,29 @@ def __init__(self, pad_size, input_size):
                 size of the original signal
         """
         self.pad_size = pad_size
+        self.input_size = input_size
 
     def __call__(self, x):
+        pad_size_tmp = list(self.pad_size)
```

Comment:
Let's avoid things like `tmp`. What distinguished this `pad_size` from `self.pad_size`?

Reply:
tmp is a list. the code breaks if self.pad_size is a list.